### PR TITLE
Create fields for Overview and Coverage on PDP and inline PurchaseForm

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
First step in towards new layout
Create fields for Overview and Coverage on PDP and inline PurchaseForm

## Justify why they are needed
Too support updated layout for PDP

https://user-images.githubusercontent.com/6661511/203581427-c6524252-6fa7-414a-9b6a-6ba2d10ba3c0.mov

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
Current dependencies on/for this PR:
* main
  * **PR #923** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/923" target="_blank">Graphite</a>  👈
    * **PR #924** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/924" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/923?utm_source=stack-comment).

File: apps/store/src/blocks/ProductPageBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+type ProductPageBlockProps = SbBaseBlockProps<{
+  overview: SbBlokData[]
+  coverage: SbBlokData[]
+  body: SbBlokData[]
+}>
+
+export const ProductPageBlock = ({ blok }: ProductPageBlockProps) => {
+  return (
+    <Main {...storyblokEditable(blok)}>
+      <PurchaseForm />
```
Comment (alebedev) [inline apps/store/src/blocks/ProductPageBlock.tsx:15]: Let's delete PriceCalculatorBlock, we no longer need it

**Review by alebedev — APPROVED** [review]
Looks good, one small comment

**Comment by gustaveen** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/923?utm_source=stack-comment).


Job Reference: JeC9qaAfvgPRoKcv6BUS

**Pull request merged**